flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/nodes/physical/common/CommonPhysicalLookupJoin.scala

```diff
@@ -190,13 +191,18 @@ abstract class CommonPhysicalLookupJoin(
       case t: TableSourceTable => t.contextResolvedTable.getIdentifier
       case t: LegacyTableSourceTable[_] => t.tableIdentifier
     }
+    val filterPushdownString: String = temporalTable match {
+      case t: TableSourceTable => getTableFilterString(t)
+      case _: LegacyTableSourceTable[_] => ""
+    }
 
     super
       .explainTerms(pw)
       .item("table", tableIdentifier.asSummaryString())
```

Comment:
What about reusing this item `table` with:
```
val tableDesc: String = temporalTable match {
  case t: TableSourceTable => t.getQualifiedName.asScala.mkString(", ")
  case t: LegacyTableSourceTable[_] => t.tableIdentifier.asSummaryString()
}

super
  .explainTerms(pw)
  .item("table", tableDesc)
  ...
```

Although the plans for some tests may change, this will not actually affect compatibility, and it can be aligned with the digest in `CommonPhysicalTableSourceScan`.